torch/testing/_internal/common_optimizers.py

```diff
@@ -1747,10 +1684,10 @@ def _get_optim_inputs_including_global_cliquey_kwargs(
             ),
             DecorateInfo(
                 skipIfTorchDynamo(
-                    "fails, https://github.com/pytorch/pytorch/issues/117165"
+                    "See https://github.com/pytorch/pytorch/issues/115679"
                 ),
                 "TestOptimRenewed",
-                "test_deepcopy_copies_all_public_attrs",
+                "test_state_dict_with_cuda_params",
```

Comment:
Is this change intentional? It looks diff from the rest

Reply:
Yeah this is intentional, it will be able to be removed after we determine the non capturable issue.

Reply:
But does this belong to this commit on the stack? i.e., does this commit cause this test_state_dict_with_cuda_params to fail now? If not, it should go elsewhere

Reply:
Removed